core/src/env/srml/srml_only/impls.rs

```diff
@@ -77,33 +67,37 @@ where
     unsafe fn load(key: Key) -> Option<Vec<u8>> {
         const SUCCESS: u32 = 0;
         if sys::ext_get_storage(key.as_bytes().as_ptr() as u32) == SUCCESS {
-            return Some(Self::read_scratch_buffer())
+            return Some(read_scratch_buffer())
         }
         None
     }
 }
 
-impl<T: EnvTypes> SrmlEnv<T> {
-    /// Load the contents of the scratch buffer
-    fn read_scratch_buffer() -> Vec<u8> {
-        let size = unsafe { sys::ext_scratch_size() };
-        let mut value = Vec::new();
-        if size > 0 {
-            value.resize(size as usize, 0);
-            unsafe {
-                sys::ext_scratch_copy(value.as_mut_ptr() as u32, 0, size);
-            }
-        }
-        value
-    }
+/// The SRML contracts environment.
+#[allow(unused)]
```

Comment:
Please do a `#[cfg_attr(...)]` here in order to do this conditionally.

Reply:
This was redundant so I've removed it